pkgs/development/python-modules/drawille/default.nix

```
@@ -0,0 +1,19 @@
+{lib, buildPythonPackage, fetchPypi}:
+
+buildPythonPackage rec {
+  pname = "drawille";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-t4nS8TWbEGKHibIbLfZZycPQxTiEzuJ7DYsa6Twi+8s=";
+  };
+
+  meta = with lib; {
+    description = "Drawing in terminal with unicode braille characters";
+    homepage = "https://github.com/asciimoo/drawille";
+    license = licenses.agpl3;
```

```suggestion
    license = licenses.agpl3Plus;
```

`agpl3` is deprecated for `agpl3Only` and `agpl3Plus`. This package specifies `agpl3Plus`: https://github.com/asciimoo/drawille/blob/master/setup.py#L9